.github/ISSUE_TEMPLATE/default.md

```diff
@@ -0,0 +1,13 @@
+## Problem to solve
+
+<!-- Explain the problem in your own words. -->
+
+## Proposal
+
+<!-- Share the proposed changes, or go ahead with creating a new merge request and linking it here. -->
+
+## Resources
+
+## Tasks
+
+- [ ] ...
```

Comment:
```suggestion
- [ ] task 1
- [ ] task 2
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Thanks, great suggestion :)